test-specifications/token-service/tokenMintTransaction.md

```diff
@@ -0,0 +1,200 @@
+
+#### JSON Request Example
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 64362,
+  "method": "mintToken",
+  "params": {
+    "tokenId": "0.0.15432",
+    "amount": "1000"
+  }
+}
+```
+
+#### JSON Response Example
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 64362,
+  "result": {
+    "status": "SUCCESS",
+    "newTotalSupply": "1001000"
+  }
+}
+```
+
+### **Amount:**
+
+- The amount of fungible tokens to mint.
+
+| Test no | Name                                                                           | Input                                                                                                                                     | Expected response                                                                                               | Implemented (Y/N) |
+|---------|--------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------|-------------------|
+| 1       | Mints an amount of 1,000,000 fungible tokens                                   | tokenId=<CREATED_FUNGIBLE_TOKEN_ID>, amount="1000000", commonTransactionParams.signers=[<CREATED_FUNGIBLE_TOKEN_SUPPLY_KEY>]              | The token mint succeeds and the token's treasury account contains 1,000,000 more minted tokens.                 | N                 |
+| 2       | Mints an amount of 0 fungible tokens                                           | tokenId=<CREATED_FUNGIBLE_TOKEN_ID>, amount="0", commonTransactionParams.signers=[<CREATED_FUNGIBLE_TOKEN_SUPPLY_KEY>]                    | The token mint succeeds and the token's treasury account contains 0 more minted tokens.                         | N                 |
+| 3       | Mints no fungible tokens                                                       | tokenId=<CREATED_FUNGIBLE_TOKEN_ID>, commonTransactionParams.signers=[<CREATED_FUNGIBLE_TOKEN_SUPPLY_KEY>]                                | The token mint succeeds and the token's treasury account contains 0 more minted tokens.                         | N                 |
+| 4       | Mints an amount of 9,223,372,036,854,775,806 (int64 max - 1) fungible tokens   | tokenId=<CREATED_FUNGIBLE_TOKEN_ID>, amount="9223372036854775806", commonTransactionParams.signers=[<CREATED_FUNGIBLE_TOKEN_SUPPLY_KEY>]  | The token mint succeeds and the token's treasury account contains 9,223,372,036,854,775,806 more minted tokens. | N                 |
+| 5       | Mints an amount of 9,223,372,036,854,775,807 (int64 max) fungible tokens       | tokenId=<CREATED_FUNGIBLE_TOKEN_ID>, amount="9223372036854775807", commonTransactionParams.signers=[<CREATED_FUNGIBLE_TOKEN_SUPPLY_KEY>]  | The token mint succeeds and the token's treasury account contains 9,223,372,036,854,775,807 more minted tokens. | N                 |
```

Comment:
Do we want to note here that the created fungible token must have `0` tokens and decimals `0` for this to succeed?

Reply:
I added that the created fungible token for each test should be created with 0 decimals.